examples/chart/teleport-demo/docker/cloudflare-agent/build.sh

```diff
@@ -1,15 +1,29 @@
-#!/usr/bin/env bash
+#!/bin/bash
 set -e
+ONE=$1
+TWO=$2
 VERSION=3.2.0
 if [[ "$1" != "" ]]; then
     VERSION=$1
     shift
 fi
+GCPROJECT=kubeadm-167321
+if [[ "" != "${TWO}" ]]; then
+    GCPROJECT=${TWO}
+    shift
+fi
+
 docker pull quay.io/gravitational/debian-grande:buster
+echo "docker build --pull \
+    -t gcr.io/${GCPROJECT}/cloudflare-agent:${VERSION} \
+    -t gcr.io/${GCPROJECT}/cloudflare-agent:latest \
+    --cache-from quay.io/gravitational/debian-grande:buster,gcr.io/${GCPROJECT}/cloudflare-agent:latest \
+    .  "
```

Comment:
Remove the echo; you can run the scripts with `bash -x` to see what it's actually doing if needed.